app/src/main/java/com/github/onlynotesswent/ui/overview/Overview.kt

```diff
@@ -126,18 +127,30 @@ fun OverviewScreen(
             selectedItem = navigationActions.currentRoute())
       }) { paddingValues ->
         Column(modifier = Modifier.fillMaxSize().padding(paddingValues)) {
-          SingleChoiceSegmentedButtonRow(modifier = Modifier.fillMaxWidth()) {
-            pageLabels.forEachIndexed { index, label ->
-              SegmentedButton(
-                  selected = pagerState.currentPage == index,
-                  shape = SegmentedButtonDefaults.itemShape(index = index, count = pageLabels.size),
-                  label = { Text(label) },
-                  onClick = {
-                    // Animate to the selected page when clicked
-                    coroutineScope.launch { pagerState.animateScrollToPage(index) }
-                  })
-            }
-          }
+          SingleChoiceSegmentedButtonRow(
+              modifier =
+                  Modifier.fillMaxWidth(fraction = 0.8f).align(Alignment.CenterHorizontally)) {
+                pageLabels.forEachIndexed { index, label ->
+                  SegmentedButton(
+                      selected = pagerState.currentPage == index,
+                      shape =
+                          SegmentedButtonDefaults.itemShape(
+                              index = index,
+                              count = pageLabels.size,
+                              baseShape = RoundedCornerShape(10)),
+                      border = ButtonDefaults.outlinedButtonBorder(false),
+                      colors =
+                          SegmentedButtonDefaults.colors(
+                              activeContainerColor = MaterialTheme.colorScheme.primaryContainer,
+                              inactiveContainerColor =
+                                  MaterialTheme.colorScheme.secondaryContainer),
```

Comment:
Also we would have to change the secondary color but I will do it in the UI-refinements pr